HISTORY.md

```diff
@@ -17,6 +17,7 @@
 
 ### Public API change
 * Remove obsolete implementation details FullKey and ParseFullKey from public API
+* When options.ttl is used with leveled compaction with compactino priority kMinOverlappingRatio, files excceeding half of TTL value will be prioritized more, so that by the time TTL is reached, few extra compactions will be scheduled to cleared them up.
```

Comment:
typo:
```suggestion
* When options.ttl is used with leveled compaction with compaction priority kMinOverlappingRatio, files exceeding half of TTL value will be prioritized more, so that by the time TTL is reached, fewer extra compactions will be scheduled to clear them up.
```